website/docs/ref/enums.doc.js

```diff
@@ -0,0 +1,78 @@
+/* @flow */
+/*
+---
+id: enums
+title: Enums
+permalink: /docs/enums.html
+prev: react.html
```

Comment:
Wait, maybe I misunderstood. Did you mean a single page for all the utility types, or one for each?